src/openms/source/KERNEL/MSExperiment.cpp

```diff
@@ -145,6 +149,306 @@ namespace OpenMS
     return ConstAreaIterator();
   }
 
+  MSExperiment::CoordinateType MSExperiment::aggregate(ConstAreaIterator begin, ConstAreaIterator end, AggregatorFunc rt_agg, AggregatorFunc mz_agg) const
+  {
+    /*std::vector<MSExperiment::CoordinateType> ity;
```

Comment:
Consider removing or uncommenting the commented-out sections of code. If they are necessary for future development, add a comment explaining their purpose. [important]